### DISCUSSION TITLE
Can scheduler instances be autoscaled on k8s?

### DISCUSSION THREAD

**Discussion by orphefs** [apache/airflow#41442] [Q&A] [answered]
I know that worker pods can use KEDA for autoscaling on Kubernetes, but is the same possible for scheduler pods? If it is, would someone be able to point me to the relevant configuration in the helm chart?

**Comment by potiuk** [comment, marked as answer]
No. No configuration there. You could however contribute such things if you would like to - following KEDA for celery - you would just need to figure out the right scaling rules
